mmv1/templates/terraform/custom_expand/enum_bool.go.erb

```diff
@@ -0,0 +1,25 @@
+<%- # the license inside this block applies to this file
+	# Copyright 2021 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d tpgresource.TerraformResourceData, config *transport_tpg.Config) (interface{}, error) {
```

Comment:
Why do we need this? Is the issue that the current resource accepts any format of `true` `TRUE` or `tRuE` and sends them to the API as `TRUE` and we need to replicate that?

Reply:
This is needed because the field is a boolean but omitting it is distinct from setting it to `false`, so we use an enum to represent it in terraform. The enum needs to be converted to a boolean before being sent to the api.